### ISSUE TITLE
EvenHubs perf tests fail to build

### ISSUE THREAD

**Issue by pakrym** [Azure/azure-sdk-for-net#26236]
Repro:
`dotnet build .\eng\service.proj`

```
D:\github\azure\net\sdk\eventhub\Azure.Messaging.EventHubs.Processor\perf\Program.cs(22,43): error CS1061: 'EventHubsTestEnvironment' does not contain a definition for 'ShouldRemoveNamespaceAfterTestRunCompletion' and no accessible extension method 'ShouldRemoveNamespaceAfterTestRunCompletion' accepting a first argument of type 'EventHubsTestEnvironment' could be found (are you missing a using directive or an assembly reference?) [D:\github\azure\net\sdk\eventhub\Azure.Messaging.EventHubs.Processor\perf\Azure.Messaging.EventHubs.Processor.Perf.csproj]
D:\github\azure\net\sdk\eventhub\Azure.Messaging.EventHubs.Processor\perf\Program.cs(22,43): error CS1061: 'EventHubsTestEnvironment' does not contain a definition for 'ShouldRemoveNamespaceAfterTestRunCompletion' and no accessible extension method 'ShouldRemoveNamespaceAfterTestRunCompletion' accepting a first argument of type 'EventHubsTestEnvironment' could be found (are you missing a using directive or an assembly reference?) [D:\github\azure\net\sdk\eventhub\Azure.Messaging.EventHubs.Processor\perf\Azure.Messaging.EventHubs.Processor.Perf.csproj]
D:\github\azure\net\sdk\eventhub\Azure.Messaging.EventHubs.Processor\perf\Program.cs(22,43): error CS1061: 'EventHubsTestEnvironment' does not contain a definition for 'ShouldRemoveNamespaceAfterTestRunCompletion' and no accessible extension method 'ShouldRemoveNamespaceAfterTestRunCompletion' accepting a first argument of type 'EventHubsTestEnvironment' could be found (are you missing a using directive or an assembly reference?) [D:\github\azure\net\sdk\eventhub\Azure.Messaging.EventHubs.Processor\perf\Azure.Messaging.EventHubs.Processor.Perf.csproj]
```

We should also fix our CI pipelines to build perf projects.

**Comment by jsquire** [comment]
Apologies; my understanding was that this work was already completed.   I'll get this patched today.

**Comment by jsquire** [comment]
Please ensure that you're rebased to the latest in `main`.   The offending lines were fixed as part of commit [0af83e4e121a5aba44f3a2c14a3eb8478f9ce661#diff-423150fb657a5db4380672600162b9c0017a0da5717c8f056e32df9eb70f5f25](https://github.com/Azure/azure-sdk-for-net/commit/0af83e4e121a5aba44f3a2c14a3eb8478f9ce661#diff-423150fb657a5db4380672600162b9c0017a0da5717c8f056e32df9eb70f5f25) in early November.

**Comment by jsquire** [comment]
I see what happened.   A new project was added for the Processor performance tests, which wasn't added to the Event Hubs solution, so it was missed when updating the test environment.     Fixed by #26238.
